clusters/ci-scheduling-webhook/dns.yaml

```diff
@@ -0,0 +1,19 @@
+# https://github.com/openshift/enhancements/blob/master/enhancements/dns/configurable-dns-pod-placement.md#proposal
+# dns-default-* pods do not tolerate all taints. This means our infra nodes and nodes tainted to support
+# pod classes are not getting these pods scheduled onto them.
+# This seemed to lead to https://bugzilla.redhat.com/show_bug.cgi?id=2086887 .
+apiVersion: operator.openshift.io/v1
```

Comment:
Hi, this is breaking CI for BM QE when resolving names as conflicting with https://github.com/openshift/release/blob/master/clusters/build-clusters/build05/assets/dns.yaml

Can we merge them for build05?